Description: Disable the tests that require rubocop, which is not yet packaged.
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2016-01-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/spec/raven/integrations/rake_spec.rb
+++ b/spec/raven/integrations/rake_spec.rb
@@ -1,7 +1,7 @@
-require 'rake'
+#require 'rake'
 
-describe 'Rake tasks' do
-  it 'should capture exceptions in Rake tasks' do
-    expect(`cd spec/support && bundle exec rake raise_exception 2>&1`).to match(/Sending event/)
-  end
-end
+#describe 'Rake tasks' do
+  #it 'should capture exceptions in Rake tasks' do
+    #expect(`cd spec/support && bundle exec rake raise_exception 2>&1`).to match(/Sending event/)
+  #end
+#end
